config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  root 'questions#index'
+
+  resources :questions, :answers, :responses
```

Comment:
I wouldn't inline these, as you will almost always want to use `only` or `except` to restrict access to the routes that you accept for each resource.